README.md

```diff
@@ -12,7 +12,7 @@ You can easily reference Cake.ExtendedNuGet directly in your build script via a
 
 Please visit the Cake Documentation for a list of available aliases:
 
-http://cakebuild.net/dsl/file-helpers
+http://localhost:49872/dsl/nuget
```

Comment:
Yes, apologies.  This was a mistake on my part.  I copied/pasted the URL from the locally running website out of Visual Studio, rather than simply alter the latter part of the URL.  Sorry.